10352.xml

```diff
@@ -0,0 +1,114 @@
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>Thermal Imager</Name>
+    <Description1><![CDATA[This LWM2M Object includes the Thermal Imager information.]]></Description1>
+    <ObjectID>10352</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10352</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Single</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+		<!--Item ID="5907">
+			<Name>Host Device Unique ID</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Optional</Mandatory>
+			<Type>String</Type>
+			<RangeEnumeration/>
+			<Units/>
+			<Description>
+				<![CDATA[ The host device unique ID as assigned by an OEM, MNO, 
+				or other as the Device ID in the onboarding or manufacturing process. ]]>
+			</Description>
+		</Item>
+		<Item ID="5905">
+			<Name>Host Device Manufacturer</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Optional</Mandatory>
+			<Type>String</Type>
+			<RangeEnumeration/>
+			<Units/>
+			<Description>
+			<![CDATA[ Human readable host device manufacturer name ]]>
+			</Description>
+		</Item>
+		<Item ID="5906">
+			<Name>Host Device Model Number</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Optional</Mandatory>
+			<Type>String</Type>
+			<RangeEnumeration/>
+			<Units/>
+			<Description>
+				<![CDATA[A host device model identifier (manufacturer specified string)]]>
+			</Description>
+		</Item-->
+		<Item ID="1">
+			<Name>Highest Temperature</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>Float</Type>
+			<RangeEnumeration>-100.0..100.0</RangeEnumeration>
+			<Units>°C</Units>
+			<Description>The Highest Temperature of the thermal imager.</Description>
+		</Item>
+		<Item ID="2">
+			<Name>Lowest Temperature</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>Float</Type>
+			<RangeEnumeration>-100.0..100.0</RangeEnumeration>
+			<Units>°C</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
°C, Celsius scale,  the unit of Temperature.

Reply:
That's "Cel" in SenML units

Reply:
Thanks for your comments, changed from °C to Cel.